models/place.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python3
-"""Place Module"""
+"""This module creates a Place class"""
```

Comment:
I think it was fine as it was before, short and straight to the point, please undo